R/dm-from-src.R

```diff
@@ -38,7 +38,7 @@
 #' dm_from_src(con)
 #'
 #' DBI::dbDisconnect(con)
-dm_from_src <- function(src = NULL, table_names = NULL, learn_keys = NULL,
+dm_from_src <- function(src = NULL, table_names = NULL, learn_keys = NULL, schema = NULL,
```

Comment:
I'd rather leave out the formal argument for now, and pass it through the ellipsis the same way as with Postgres. This will make the pull request much smaller.

```suggestion
dm_from_src <- function(src = NULL, table_names = NULL, learn_keys = NULL,
```